pkg/api/handler.go

```diff
@@ -88,14 +88,14 @@ func (h *handlerState) execute(ctx context.Context, tracker attribute.Tracker, a
 	// get a new context with the attribute bag attached
 	ctx = attribute.NewContext(ctx, ab)
 
-	untypedCfg := h.cfg.Load()
-	if untypedCfg == nil {
-		// config has NOT been loaded yet
-		const msg = "Configuration is not available"
+	cfg := h.cfg.Load().(config.Resolver)
```

Comment:
trying to type assert nil to anything results in panic.
https://play.golang.org/p/59t2a3-Tsf
`panic: interface conversion: interface {} is nil, not config.Resolve`

That is why we need to fetch untypedCfg first.